scripts/cibuild/ConfigureVstsBuild.ps1

```diff
@@ -65,10 +65,11 @@ Function Update-VsixVersion {
 
 Function Set-RtmLabel {
     param(
-        [string]$BuildRTM
+        [Parameter(Mandatory = $true)]
+        [boolean]$isRTMBuild
```

Comment:
Consider `Switch` instead.

Reply:
Looking at [the docs on switch parameters](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/types-of-cmdlet-parameters?view=powershell-7.1#switch-parameters), I don't really like it.

As a boolean parameter, we can parse the string as a bool in the outer function, and then pass the strongly typed value into the inner function, which we can then use in a condition.

If we want to use a switch parameter, then the outer function needs to condition whether or not it passes the switch, as well as the inner function having a if-else block.

I think it just adds more branching.

Reply:
I disagree, but I don't care too much :D